### PR TITLE
Add orange3-ImageAnalytics

### PR DESCRIPTION
**Pull request by PrimozGodec**

### CODE REVIEW

File: recipes/orange3-imageanalytics/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    Orange3 Image Analytics is an add-on for the Orange3 data mining suite.
+    It provides extensions for importing/creating labeled image data sets and
+    embedding them through a variety of pre-trained deep neural networks.
+  doc_url: http://orange3-imageanalytics.readthedocs.io
+  dev_url: https://github.com/biolab/orange3-imageanalytics
+
+extra:
+  recipe-maintainers:
+    - primozgodec
+    - astaric
```
Comment (astaric) [inline recipes/orange3-imageanalytics/meta.yaml]: I agree with being a maintainer

File: recipes/orange3-imageanalytics/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (dougalsutherland) [inline recipes/orange3-imageanalytics/meta.yaml]: Think this one can be `noarch:python`.

Comment (dougalsutherland) [inline recipes/orange3-imageanalytics/meta.yaml]: You probably also need `preserve_egg_dir: True` for the entry points to work correctly; I assume this is how Orange3 knows about the plugin.

Comment (PrimozGodec) [reply recipes/orange3-imageanalytics/meta.yaml]: Orange can access documentation online so it is fine.

Comment (PrimozGodec) [reply recipes/orange3-imageanalytics/meta.yaml]: It can't be noarch: python since skip parameter is present (this addon is python3 only)

**Comment by dougalsutherland** [comment]
Rebased for you. Was probably premature since `hyper` builds aren't actually up yet, though; will hit rebuild in a bit.

**Comment by dougalsutherland** [comment]
I don't know why CircleCI ended up in the checks twice, but it and Appveyor both passed, and Mac passed locally + it's pure python, so I'm merging. Thanks @PrimozGodec!

**Comment by PrimozGodec** [comment]
@dougalsutherland sorry for bothering you again, but do you have an idea why hyper not found? It exists on [anaconda cloud](https://anaconda.org/conda-forge/hyper) but I can not even install it locally using 

    conda install -c conda-forge hyper 

I get `PackageNotFoundError` on my MacOS.

**Comment by dougalsutherland** [comment]
Looks like it's still not built for Mac: https://anaconda.org/conda-forge/hyper/files only shows Linux / Windows. Travis will build it, eventually; [here's the page](https://travis-ci.org/conda-forge/hyper-feedstock/builds/328170556) where it's queued. Travis was having issues with their Mac infrastructure today ([status page](https://www.traviscistatus.com/)) and they have a huge backlog right now.

**Pull request merged**